src/main/java/frc/robot/subsystems/climb/ClimbArm.java

```diff
@@ -0,0 +1,70 @@
+package frc.robot.subsystems.climb;
+
+import static frc.robot.Constants.ClimbConstants.*;
+
+import java.util.function.Consumer;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkBase.SoftLimitDirection;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.util.MotorUtil;
+
+public class ClimbArm {
+    private final CANSparkMax winchMotor;
+    private RelativeEncoder extensionEncoder;
+
+    private final DigitalInput zeroLimitSwitch;
+
+    private final Consumer<CANSparkMax> configureMotor = sparkMax -> {
+        sparkMax.setIdleMode(IdleMode.kBrake); 
+        sparkMax.setInverted(false);
+
+        extensionEncoder = sparkMax.getEncoder();
+        extensionEncoder.setPositionConversionFactor(Units.inchesToMeters(AXLE_RADIUS_INCHES) * 2 * Math.PI / WINCH_REDUCTION);
```

Comment:
I dont think you can use this theoretical value, the axle is a hex axle, so the radius is not always .5in, and the string's width is not negligible. We can use this value as a good starting place, but it will be more accurate if it is found through testing.